extensions/2.0/Khronos/KHR_audio/schema/audio.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Audio Source",
+  "type": "object",
+  "description": "A source for audio data referenced in Audio Emitters Audio can be referenced by URI or `bufferView` index. `mimeType` is required when `bufferView` is used.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "uri": {
+          "type": "string",
+          "description": "The uri of the audio file.",
+          "format": "uriref",
+          "gltf_detailedDescription": "The uri of the audio file. Relative paths are relative to the .gltf file.",
+          "gltf_uriType": "audio"
+      },
+      "mimeType": {
+          "type": "string",
+          "description": "The audio's MIME type. Required if `bufferView` is defined. Unless specified by another extension, the only supported mimeType is `audio/mpeg`.",
+          "anyOf": [
+                {
+                    "enum": [ "audio/mpeg" ]
+                },
+                {
+                    "type": "string"
```

Comment:
We should consider supporting more formats than just MP3. For example, WAV files are preferred for short sound effects due to their rapid decode time. There's also OGG vorbis which is a superior format for the same use cases as MP3. There's a long list of other formats but I don't think we should try to support all of them, maybe just 2 or 3.

Reply:
todo: @antpb update this description to include wav, mp3, and other supported types that are commonly implemented in current engines. state that others are nonstandard. Set type to json instead of string: match https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/schema/image.schema.json#L16C8-L27C15

Reply:
oh, dont forget OGG! Support in Safari for OGG was blocking adding it when we first drafted the OMI_audio_emitter spec. It has since been added later in 2021

Reply:
Would be nice to have FLAC added, too.

Reply:
FLAC is a great format, but I don't think it's the right fit for the base glTF audio spec. FLAC is supported on all browsers since 2017, and in Unreal, but not Unity or Godot. FLAC is great for lossless audio, but glTF is generally focused on being a last mile format that focuses on being easy to parse by many importers,[^1][^2] in which cases lossless audio is usually unnecessary. For improved portability, I think it's best for the base spec to only support MP3/OGG/WAV to encourage assets to use those, and then FLAC can be non-standard.

[^1]: https://github.com/KhronosGroup/glTF/issues/1051#issuecomment-544647316
[^2]: https://nickporcino.com/posts/last_mile_interchange.html

Reply:
When referring to Ogg audio, Vorbis codec is usually implied as Ogg is a just container format. Ogg format is not supported in Safari so it won't be added to this extension.

Reply:
Yeah the main issue here is finding formats with widespread support. By listing a format here, it is mandatory to implement that codec in order to support this glTF extension.
The Safari columns here show the problem: https://caniuse.com/?search=ogg

I would like to say, however, that I don't see any requirement for a given browser rendering engine to support a particular format. As with other glTF extensions, when some feature is not natively used by a particular game engine or renderer, it is possible to transcode at load time.

In the case of Ogg Vorbis, it would be possible to transcode Ogg Vorbis to another supported audio format using widely available open code such as https://github.com/brion/ogv.js/ - would usage of a library like Ogv.js in WASM not address the possible concern of Safari support?

Reply:
> As with other glTF extensions, when some feature is not natively used by a particular game engine or renderer, it is possible to transcode at load time.

The only current examples are Draco, Meshopt, and Basis compression codecs. They always have to implemented by engines (likely with the same open-source libraries) and do not depend on browser support.

That said, Safari does support Vorbis audio inside WebM files, so using the latter instead of Ogg could be explored.

Reply:
Originally when this PR spec was written we opted to only define mp3 in the base spec with any other types being "non-standard" but still capable of being used through a string definition of the MIME desired. Mainly because of the expectation of every implementation needing to support every type defined in this base spec. We established that the majority of potential implementors had a common thread of supporting mp3 so everything else would be non-standard and most compatible going forward.

My comment earlier in this was about adding wav because of the wide usage for things like sound effects and making this more in line with the MSFT spec that *only* supported wav. OGG has a similar situation with wide usage, but we get into a long list of MIMEs with no real criteria for what should make the list of base spec supported types.

Being that this list could be ever-dynamic we should maybe agree on the base list of types with precedent in specification (wav/mp3) agreeing that anything else is non-standard and allowed. This puts us in a situation where any file type can be used and common types implemented (maybe FLAC) will be supported and parsed by engines regardless of the base spec. 

It feels unrealistic to expect every implementation to have to support every file type that we list here and the list feels fragile to be ever changing with many opinions of what should be supported. 

TLDR, can we go `[ "audio/wav", "audio/mpeg" ]` with the spec allowing any other string being added as part of the array for further supports?